wlm_server/task/handler.py

```diff
@@ -21,12 +24,58 @@ class TaskHandler(threading.Thread):
 
     def __init__(self):
         super().__init__()
+        self._wlm: WLM
         self._message_queue: MessageQueue = settings.MESSAGE_QUEUE
         self._measure_queue: MeasureQueue = MeasureQueue()
         self._channel_to_period: dict[int, timedelta] = {}
+        self._open_connection()
+
+    def _open_connection(self):
+        config = Config.objects.first()
+        wlm_version = config.wlm_version
+        wlm_dll_path = config.wlm_dll_path
+        wlm_app_path = config.wlm_app_path
+        self._wlm = WLM(wlm_version, wlm_dll_path, wlm_app_path)
+        self._wlm.open()
+        self._wlm.start_measurement()
+
+    def _close_connection(self):
+        self._wlm.stop_measurement()
+        self._wlm.close()
+
+    def _start_channel_measurement(self, channel: int):
+        setting = Setting.objects.filter(channel__name=channel).order_by('-created_at').first()
```

Comment:
Let's keep this in mind: the filtering might cost quite a lot in the future.
For now I think it would be fine.